src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -816,4 +816,35 @@ public static function createCache($namespace, $defaultLifetime, $version, Logge
 
         return $apcu;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPropertyValue($object, string $property)
+    {
+        $zval = [
+            self::VALUE => $object,
+        ];
+
+        return $this->readProperty($zval, $property)[self::VALUE];
```

Comment:
@fbourigault i am not sure with this. But why is here no catch unlike in case of setPropertyValue ?

Both can throw and have invalid arguments?